infra/aws/terraform/modules/sns-via-email/main.tf

```diff
@@ -0,0 +1,28 @@
+resource "aws_sns_topic" "this" {
+  name = var.name
+  delivery_policy = jsonencode({
```

Comment:
This will enforce the policy for every SNS topic and don't give room for customization. Can use pass the path of a json file to the variable ?